WordPress/src/main/java/org/wordpress/android/util/LogFileProviderWrapper.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import org.wordpress.android.util.helpers.logfile.LogFileProvider
+import javax.inject.Inject
+
+class LogFileProviderWrapper @Inject constructor(context: Context) {
+    val logFileProvider = LogFileProvider.fromContext(context)
```

Comment:
This can probably be private?

```suggestion
    private val logFileProvider = LogFileProvider.fromContext(context)
```

Reply:
True, fixed in a180062769ac8b6ffb1e37fba808e84bf0504185